woocommerce-services.php

```diff
@@ -819,6 +822,74 @@ public function deactivate_notice() {
 			<?php
 		}
 
+		public function check_jetpack_install() {
+			if ( defined( 'JETPACK_DEV_DEBUG' ) ) {
+				return true;
+			}
+
+			if ( ! class_exists( 'Jetpack_Data' ) ) {
+				return false;
+			}
+
+			$user_token = Jetpack_Data::get_access_token( JETPACK_MASTER_USER );
+			return $user_token && is_object( $user_token ) && isset( $user_token->external_user_id );
+		}
+
+		public function admin_jetpack_notice() {
+			wp_enqueue_style( 'wc_connect_banner' ); // Including the CSS this early will mostly prevent a flash of Core styles in the banner
+			add_action( 'admin_notices', array( $this, 'show_jetpack_notice' ) );
+		}
+
+		public function show_jetpack_notice() {
+			$button_onclick = '';
+			$button_url = '#';
+
+			if ( class_exists( 'Jetpack_Data' ) ) {
+				$screen = get_current_screen();
+				if ( 'jetpack' === $screen->parent_base || 'plugins' === $screen->base ) {
+					return; // Jetpack Dashboard and the plugins list screen already have big "Connect to WordPress.com" buttons
+				}
+
+				$notice_text = __( 'To get started please connect Jetpack to your Worpress.com account.', 'woocommerce-services' );
```

Comment:
Typo :)

Let's go with: `'To get started, please connect Jetpack to your WordPress.com account.'`